### PR TITLE
fixed ReadMe.md

### PR DESCRIPTION
**Pull request by manojuppala**
Fixed some formatting issues in readme

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -104,17 +104,20 @@ Contributions are what make the open source community such an amazing place to b
...
 
 [![portfolio](https://img.shields.io/badge/my_portfolio-000?style=for-the-badge&logo=ko-fi&logoColor=white)](https://raghavyuva.com/)
 [![linkedin](https://img.shields.io/badge/linkedin-0A66C2?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/raghavyuva)
 [![twitter](https://img.shields.io/badge/twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white)](https://twitter.com/yuva_raghav)
 
 ## Donate
+
 If you find this project as useful then do donate so that the project is maintained up-to date.
-use  theis platforms for donation.
+use theis platforms for donation.
```
Comment (raghavyuva) [inline ReadMe.md:121]: Use this , Make Changes here

Comment (manojuppala) [reply ReadMe.md]: fixed

Comment (raghavyuva) [reply ReadMe.md]: Remove unnecessary commits and make a single commit,

Comment (manojuppala) [reply ReadMe.md]: done

**Review by raghavyuva — APPROVED** [review]

**Pull request merged**